packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -98,43 +95,20 @@ const Updates = decorate([
         const { effects, state } = this
         await xoaUpdater.configure({
           ...pick(state, [
+            'channel',
```

Comment:
Why not use `_channel`?

Reply:
Because if the user don't interact with the UI, the correct value will be the props value.